eventuals/builder.h

```diff
@@ -117,6 +123,12 @@ class FieldWithDefault<Value_, true> : public Field<Value_, true> {
     : Field<Value_, true>(std::forward<Value>(value)) {}
 
   FieldWithDefault(FieldWithDefault&&) = default;
+
+  template <typename Value>
+  auto Set(Value&& value) {
+    static_assert(std::is_convertible_v<Value, Value_>);
+    return FieldWithDefault<Value_, true>{std::forward<Value>(value)};
+  }
```

Comment:
Apparently, we could not use multiple Request::_Builder::header() before, so I had to do this change. I did the same for `Field<Value_, true>` as well for consistency.

Reply:
I personally think going with just `Headers` type is the way to handle such cases, dunno if we really want to wrap things up.